kernel/src/log_segment.rs

```diff
@@ -32,8 +32,8 @@ use url::Url;
 pub(crate) struct LogSegment {
     pub end_version: Version,
     pub log_root: Url,
-    /// Sorted commit files in the log segment
-    pub commit_files: Vec<ParsedLogPath>,
+    /// Sorted commit files in the log segment (ascending)
+    pub sorted_commit_files: Vec<ParsedLogPath>,
```

Comment:
Update: yes, the action iterator was updated with a `rev` call.

Reply:
Reverse iterator is in use. I don't have a test for checking that it's reverse sorted because that would require that we `replay`. This needs a read schema, engine, and some engine data wrangling. This seems out of scope for this PR.